Frontend/ui-common/src/Components/ChooseVehicle/View.purs

```diff
@@ -28,20 +29,26 @@ view push config =
   , padding $ Padding 8 16 12 16
   , clickable config.isEnabled
   , onClick push $ const $ OnSelect config
-  , afterRender push (const NoAction)
+  , orientation VERTICAL
+  , afterRender 
+    ( \action -> do
+        _ <- pure $ spy "shrey00" config
```

Comment:
Remove logs. Also is this afterRender required?

Reply:
No its not required, removed